src/core/tests/main.cpp

```diff
@@ -97,7 +97,7 @@ GTEST_API_ int main(int argc, char **argv)
     t.join();
     
     // exit without any destruction
-    dsn_terminate();
+    //dsn_terminate();
```

Comment:
the underlying engines does not consider the dctor case currently, so elegant exit is not available right now.